descartes_moveit/src/ikfast_moveit_state_adapter.cpp

```diff
@@ -0,0 +1,155 @@
+/*
+ * Software License Agreement (Apache License)
+ *
+ * Copyright (c) 2016, Jonathan Meyer
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "descartes_moveit/ikfast_moveit_state_adapter.h"
+
+#include <cassert>
+#include <eigen_conversions/eigen_msg.h>
+#include <ros/node_handle.h>
+
+const static std::string default_base_frame = "base_link";
+const static std::string default_tool_frame = "tool0";
+
+// Compute the 'joint distance' between two poses
```

Comment:
@BrettHemes @shaun-edwards I'm rather embarrassed to admit there was no great thought put into this choice of norm for computing the distance between poses. I will at least say that, for better or worse, the L1 matches the cost function used to compute edge costs when solving our graphs.

I can change this if you think it's more correct.